packages/core/src/application/get_container_logs.rs

```diff
@@ -0,0 +1,45 @@
+use crate::infra::docker::get_docker_instance;
+use eyre::Result;
+use tokio_stream::StreamExt;
+
+pub async fn get_container_logs(
+    container_name: &str,
+    tail_lines: Option<usize>,
+) -> Result<Vec<String>> {
+    let docker = get_docker_instance()?;
+
+    let tail = tail_lines
+        .map(|n| n.to_string())
+        .unwrap_or_else(|| "all".to_string());
+
+    let options = bollard::container::LogsOptions::<String> {
+        stdout: true,
+        stderr: true,
+        follow: false,
+        timestamps: true,
+        tail,
+        ..Default::default()
+    };
+
+    let logs = docker.logs(container_name, Some(options));
```

Comment:
think it would be better to move some of this functionality into `infra/docker.rs`

Reply:
resolved [e55ec86](https://github.com/kittynode/kittynode/pull/480/commits/e55ec8684c2b9aaadbc1f0ff824e97e63c73bb2b)